### ISSUE TITLE
Dockerize

### ISSUE THREAD

**Issue by heyMP** [elmsln/wcfactory#1]
Issues:
- [ ] Docker won't npm install a package.json with template variables in them.  Need to alter package.json without fsCopyTemplate

**Comment by btopro** [comment]
make a boilerplate to do the initial install and then during a later step overwrite it w/ a templatized copy which then a generator overwrites?

**Comment by heyMP** [comment]
yes possibly.  Either that, or modify the package.json via transforming it to a javascript object, add/removing dependencies and field names, and then saving it with a standard `fs.writeFileSync()`.  Whichever you think would be easier.

**Comment by mwcz** [comment]
Since node and npm are already platform independent, what problem is being solved by Dockerizing?

**Comment by btopro** [comment]
@mwcz environment standardization specifically around training. We're going to start doing high scale (as far as class size) trainings surrounding web components and using the factory. We can't get bogged down in "oh your running node 7.x when we need 6.x or 8.x" when there's like 50-200 ppl there. We've been making docker images to skip on these dependency issues in past trainings to go from like 6-8 person scale to 15-20 and maybe end up with like 1 person who can't get docker working and have to fudge around with.

Docker won't be required, it'll just help those get on ramped, especially those in protected environments that may allow docker but not have full admin rights to install npm packages globally (gov, edu, parma)

**Comment by mwcz** [comment]
@btopro I see!  That helps, thanks.  I can definitely see the training angle.
